kodex-team-leadera.md

```diff
@@ -0,0 +1,63 @@
+# Kodex team leadera
+
+_Jednoduchá příručka, jak být dobrým team leaderem_
+
+Vést lidi není jednoduchá práce nejen v Appliftingu, někteří se s tím prostě narodí a nemusí pro to nic dělat, jiní naopak potřebují více času a trochu prostrčit. Všichni ale každopádně mají stále co zlepšovat. V tomto krátkém souhrnu je orientační mapa pro každého, kdo chce na pozici teamleada obstát. Jedná se v podstatě o guide pro teamleady.
```

Comment:
Myslím, že "někdo se s tím narodí a nemusí pro to nic dělat" může někoho vést k tomu, že je právě v téhle škatulce a že na sobě nemusí pracovat. Určitě jsou lidi, co pro vedení mají předpoklady, ale je IMO nutné zdůraznit, že je to úplně jiná činnost, než doteď v práci vykonávali a je třeba se jí naučit jako se učili např. programovací jazyk.

Možná zmínit, že se velmi počítají nepracovní zkušenosti s vedením lidí. Zájmové a sportovní kroužky, herní klany, učitelské zkušenosti, atd.